PalCalc.SaveReader/SaveFile/LevelMetaSaveFile.cs

```diff
@@ -26,13 +21,25 @@ public GameMeta ReadGameOptions()
         {
             var valuesVisitor = new ValueCollectingVisitor(".SaveData", ".WorldName", ".HostPlayerName", ".HostPlayerLevel");
             VisitGvas(valuesVisitor);
-
-            return new GameMeta
+            try
```

Comment:
I'd like to properly parse the `LevelMeta.sav` file instead of just try/catching this case

You can inspect the full contents + structure by setting `ARCHIVE_PRESERVE` as a compiler macro in the `PalCalc.SaveReader` project under Build settings, use `var properties = ParseGvas();` to load the full structure into memory, and set a breakpoint to inspect the properties

For values which can be simply extracted, those properties have a `Path` variable which will tell you the full path to read (first param in `ValueCollectingVisitor` is a base path, all subsequent paths are relative to that, e.g. `.SaveData.WorldName`, etc.)

What exception were you getting originally? An error due to missing values in `valuesVisitor.Result` or some underlying GVAS-read error?